### ISSUE TITLE
Metamask error with BSC integration

### ISSUE THREAD

**Issue by bhargav55** [bcnmy/metatx-standard#10]
MetaMask - RPC Error: Internal JSON-RPC error. 
{code: -32603, message: "Internal JSON-RPC error.", data: {…}}
code: -32603
data: {code: -32000, message: "**missing from address"**}

**Comment by tomarsachin2271** [comment]
Yeah this issue come from Binance testnet RPC URL. Will be fixed in next version. Will update here once this is fixed.

**Comment by nsafari** [comment]
Is there any update in this regard?
